yroots/subdivision.py

```diff
@@ -24,7 +24,7 @@
 import warnings
 
 def solve(method, funcs, a, b, rel_approx_tol=1.e-8, abs_approx_tol=1.e-12,
-          trim_zero_tol=1.e-10, max_cond_num=1e5, macaulay_zero_tol=1.e-13,
+          trim_zero_tol=1.e-10, max_cond_num=1e5,
```

Comment:
I also got rid of the Macaulay_zero_tol since we don't even use it. This will make the default tolerance search a little easier, too.